File: Assignment 7/Problem2.java
//Write a program which accept number from user and return the count of odd digits.

/*
  Input : 2395
  Output : 3
  
  Input : 1018
  Output : 2
*/

import java.util.Scanner;
public class Problem2 
{
    public static void main(String [] arg)
    {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter Number : ");
        int iNo = sc.nextInt();
        int count = 0;

        while(iNo != 0)
        {
            int rem = iNo % 10;
            if(iNo%2 != 0)
            {
                count++;
            }
            iNo /= 10;
        }
        System.out.println(count);
    }
    
}

File: Assignment 3/problem2.c
//// Write a program which accept one number from user and print even factor of that number .

// Input : 24
// Output : 1 2 4 6 8 12

#include <stdio.h>
void DisplayFactor(int iNo) // function defination
{
    int i = 0;

    if (iNo <= 0)
    {
        iNo = -iNo;
    }
    for (i = 1; i < iNo; i++)
    {
        if (iNo % i == 0)
        {
            printf("%d\t", i);
        }
    }
}
int main()
{
    int iValue = 0;

    printf("Enter number: ");
    scanf("%d", &iValue);

    DisplayFactor(iValue); // function call

    return 0;
}